src/partials/device_profile_session_id_required.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo" />
+  </div>
+  <h1 class="heading">Session ID Cookie Not Found</h1>
+  <form class="form form--margin-small" id="AuthnWidgetForm">
+    <div class="text-block">
+      <div class="icon-feedback icon-feedback--error">
+        <span class="icon-feedback__icon pingicon-ghost-error icon-feedback--error"></span>
+      </div>
+      <div class="text-block--overflow-wrap">
+        A ThreatMetrix Session ID cookie was not found. Please configure the ThreatMetrix adapter according to
```

Comment:
This state isn't specific to ThreatMetrix anymore right?

Reply:
Yes indeed. I will update.